### PR TITLE
feat: IPRPC over IBC: Part 2 - CNS-964: CLI for submitting an IPRPC over IBC proposal

### PR DESCRIPTION
**Pull request by oren-lava**
In this PR I created a helper CLI that submits a proposal which uses funds from the community pool to send an ibc-transfer TX with a custom memo that triggers an IPRPC over IBC fund request.

The help section of the CLI is extensive and includes more details and examples. The TX was tested locally successfully (submitted a proposal, made a deposit, and voted to trigger an IPRPC over IBC fund request).

### CODE REVIEW

File: x/rewards/client/cli/query_generate_ibc_iprpc_tx.go [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+			}
+			clientCtx.GenerateOnly = true
+			sender := clientCtx.FromAddress.String()
+
+			// create memo
+			senderAddr, err := utils.ParseCLIAddress(clientCtx, sender)
+			if err != nil {
+				return err
+			}
+			memo, err := createIbcIprpcMemo(senderAddr, spec, duration)
```
Comment (Yaroms) [inline x/rewards/client/cli/query_generate_ibc_iprpc_tx.go:67]: what about the amount?

File: x/rewards/client/cli/query_generate_ibc_iprpc_tx.go [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+}
+
+func createIbcIprpcMemo(creator string, spec string, duration string) (string, error) {
+	// sanity check
+	_, err := strconv.ParseUint(duration, 10, 64)
+	if err != nil {
+		return "", err
+	}
+
+	data := map[string]interface{}{
```
Comment (Yaroms) [inline x/rewards/client/cli/query_generate_ibc_iprpc_tx.go:138]: use the original struct and than marshal it

File: x/rewards/client/cli/query_generate_ibc_iprpc_tx.go (hunk 1 above)
Comment (oren-lava) [reply x/rewards/client/cli/query_generate_ibc_iprpc_tx.go:67]: the amount is not part of the memo. It's an argument to the ibc-transfer TX

File: x/rewards/client/cli/query_generate_ibc_iprpc_tx.go (hunk 2 above)
Comment (oren-lava) [reply x/rewards/client/cli/query_generate_ibc_iprpc_tx.go]: done. see https://github.com/lavanet/lava/commit/e29f0e73644f40f55eb5f8621c5a58df6cef7898

File: x/rewards/client/cli/query_generate_ibc_iprpc_tx.go [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+			if memoOnly {
+				cmd.Println(escapeMemo(memo))
+				return nil
+			}
+
+			// create ibc-transfer msg
+			if len(args) != 5 {
+				return fmt.Errorf("not enough arguments for generating TX")
+			}
+			node, err := cmd.Flags().GetString(flags.FlagNode)
```
[the remark this answers is not in the document]
Comment (oren-lava) [reply x/rewards/client/cli/query_generate_ibc_iprpc_tx.go]: done. see https://github.com/lavanet/lava/commit/e29f0e73644f40f55eb5f8621c5a58df6cef7898

**Pull request merged**